plugins/meta/aos-firewall/firewall/firewall.go

```diff
@@ -403,14 +499,43 @@ func (f *Firewall) formatIptablesRequest(chain *AccessChain) (chainFilters []ipt
 						src:      dchain.Address.IP.String(),
 						protocol: rule.Protocol,
 						dPorts:   rule.Ports,
-						jump:     "ACCEPT"})
+						jump:     "ACCEPT",
+					})
+
+					// Allow traffic forward between containers on different subnets
+					outputParams = append(outputParams, iptablesRequest{
+						chain: forwardChainName,
+						src:   dchain.Address.IP.String(),
+						dest:  chain.Address.IP.String(),
+						jump:  "ACCEPT",
+					})
+
+					outputParams = append(outputParams, iptablesRequest{
+						chain: forwardChainName,
+						src:   chain.Address.IP.String(),
+						dest:  dchain.Address.IP.String(),
+						jump:  "ACCEPT",
+					})
 				}
 			}
 		}
 	}
 
 	chainFilters = append(chainFilters, outputParams...)
 
+	_, gatewayMask, err := getInterfaceInfoByIP(chain.Gateway)
+	if err == nil {
+		chain.GatewayPrefixLen = gatewayMask
+
+		// Allow traffic forward between containers on the same subnets
+		chainFilters = append(chainFilters, iptablesRequest{
+			chain: forwardChainName,
+			src:   chain.Address.IP.String(),
+			dest:  chain.Gateway.String() + "/" + chain.GatewayPrefixLen,
```

Comment:
What drawbacks we will have without prefix len?

Reply:
in this case, we can't access another container in the same subnet